typescript-conditionals/main.ts

```diff
@@ -0,0 +1,127 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+interface Person {
+  name: string;
+  age: number;
+}
+
+function isUnderFive(number: number): boolean {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log(isUnderFive(18));
+
+function isEven(number: number): boolean {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log(isEven(5));
+
+function startsWithJ(string: string): boolean {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log(startsWithJ('Jake'));
+
+function isOldEnoughToDrink(person: Person): boolean {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const homer: Person = {
+  name: 'homer simpson',
+  age: 40,
+};
+console.log(isOldEnoughToDrink(homer));
+
+function isOldEnoughToDrive(person: Person): boolean {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const tyler: Person = {
+  name: 'tyler defiesta',
+  age: 19,
+};
+console.log(isOldEnoughToDrink(tyler));
+
+function isOldEnoughToDrinkAndDrive(person: Person): boolean {
+  if (person.age >= 0 && person.age >= 21) {
+    return false;
+  } else {
+    return true;
```

Comment:
🛑 👮‍♂️  You should never drink and drive! This should only return false. Also you have `person.age >= 21` which is the age for drinking, but can anyone over the age of 0 drive? 🤔